sim/common/itemhelpers/weaponprocs.go

```diff
@@ -70,6 +76,19 @@ func CreateWeaponProcDamage(itemId int32, itemName string, ppm float64, spellId
 	})
 }
 
+// CoH is Chance on Hit
```

Comment:
For the comments on these, could you make this something like

```
// Creates a weapon proc for "Chance on Hit" weapon effects
```

Reply:
done